docs/indepth/installation.rst

```diff
@@ -23,51 +23,53 @@ Linux
 -----
 
 Before you install ``flocker-cli`` you will need a compiler, Python 2.7, and the ``virtualenv`` Python utility installed.
-On Fedora 20 you can install these by running:
 
-.. code-block:: console
+On CentOS 7 or Fedora 20 you can install these by running:
 
-   alice@mercury:~$ sudo yum install @buildsys-build python python-devel python-virtualenv
+.. prompt:: bash linux$
```

Comment:
The prompts should be consistent throughout the docs, as much as possible. `alice@mercury` is elsewhere and seems to be the standard in our docs for commands run on a development machine.